JS-Algorithms/challenges.js

```diff
@@ -41,13 +57,36 @@ Invoking "circularArray(2)" should return "["Island", "Japan", "Israel", "German
 const COUNTRY_NAMES = ["Germany", "Norway", "Island", "Japan", "Israel"];
 
 const circularArray = (index) => {
-  // YOUR CODE HERE...
+  function resetIndex () {
```

Comment:
For naming clarity we could rename this function to `rotateIndex`.

What's the difference between the regular function (with function keyword) and an arrow function as we have in line 59?